setup.cfg

```diff
@@ -7,8 +7,8 @@ message = Bump up to version {new_version}.
 [flake8]
 filename = *.py,*.pyx,*.pxi,*.pxd
 exclude = .eggs,*.egg,build,extern,doc/source/gettingstarted/examples
-ignore = E225,E226,E227,E741,E999,W504,N
-per-file-ignores = 
+ignore = E225,E226,E227,E741,E999,W504,N,D,RST
+per-file-ignores =
```

Comment:
This removal of trailing whitespace will be reverted on the next release, because `bumpversion` always outputs trailing whitespace here...

Reply:
Actually there is an action item here, thanks for the reminder. I meant to add setup.cfg to the exclude for the trailing-whitespace hook so that we don't go through this every time.